ui/app/AppLayouts/Chat/CommunityColumn.qml

```diff
@@ -181,7 +178,6 @@ Item {
                 }
 
                 StatusMenuItem {
-                    enabled: chatsModel.communities.activeCommunity.admin
```

Comment:
Are we sure about these removals? Should anybody be able to edit an delete categories? How is that related to removing the context menu leftover?

Reply:
Ah I see, I guess this is because the menu will only be shown when the user is an admin anyways, so the condition here can be removed.

Reply:
Exactly, before the check was on each item with the same condition, that is why I was able to move the check to the parent and avoid the duplication of the check for each entry.